packages/core/core/src/RequestTracker.js

```diff
@@ -846,6 +846,8 @@ export class RequestGraph extends ContentGraph<
   }
 }
 
+const NODES_PER_BLOB = 2 ** 14;
```

Comment:
How did we arrive at this number?

Reply:
I profiled locally on my machine on how long it took to serialise `n` nodes. I tuned `n` with binary search until I reached approximately ~50 ms serialisation time per blob. The goal is to free up the event loop for a reasonable amount of time for user perception.

I'll document this on the constant so it can be tuned in the future if required.